src/Compilers/Core/Portable/Operations/OperationOld.cs

```diff
@@ -0,0 +1,35 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis
+{
```

Comment:
Consider adding a `PROTOTYPE` comment to make sure we remember to delete this before merging to master